src/zc/buildout/easy_install.py

```diff
@@ -1625,17 +1637,17 @@ class IncompatibleConstraintError(zc.buildout.UserError):
 IncompatibleVersionError = IncompatibleConstraintError # Backward compatibility
 
 
-def call_easy_install(spec, dest):
+def call_pip_install(spec, dest):
     """
-    Call `easy_install` from setuptools as a subprocess to install a
+    Call `pip install` from a subprocess to install a
     distribution specified by `spec` into `dest`.
     Returns all the paths inside `dest` created by the above.
     """
-    path = setuptools_path
+    path = pip_path
 
     args = [sys.executable, '-c',
             ('import sys; sys.path[0:0] = %r; ' % path) +
-            _easy_install_cmd, '-mZUNxd', dest]
+            _pip_install_cmd, 'install', '--no-deps', '-t', dest]
```

Comment:
I was thinking the other day (a bad habit, I know).  pip vendors a bunch of libraries (requests, urllib3, certifi, etc).  Debian doesn't allow multiple copies of a library in the distribution, so Debian patches pip to unvendor all these and makes it use their packaged versions.

I wonder, if you happen to use Debian's /usr/bin/python3 and Debian's python3-pip, will pip_path contain everything needed to run pip?

I'm guessing it will: if we use Debian's pip, then pip_path ought to contain /usr/lib/python3.X/dist-packages, which has all the needed bits.  If we don't use Debian's pip, then pip'll use its own vendored copies.

I wonder if it would be a good idea to set up a Debian-based build in the CI.  (I'm not going to insist.)

Reply:
I guess the easiest would be to run the tests in a Debian based container. is that what you mean ?

Reply:
Yes.